src/docfx/build/xref/XrefResolver.cs

```diff
@@ -224,9 +224,14 @@ private void ValidateExternalXref(IReadOnlyDictionary<string, InternalXrefSpec[]
         {
             if (!internalXrefMap.ContainsKey(xrefGroup.Key))
             {
-                _errorLog.Add(Errors.Xref.UidNotFound(
-                    xrefGroup.Key, xrefGroup.Select(xref => xref.ReferencedRepositoryUrl).Distinct(), xrefGroup.First().SchemaType) with
-                { Level = _config.IsLearn ? ErrorLevel.Error : ErrorLevel.Warning });
+                foreach (var item in xrefGroup)
```

Comment:
Do we need to distinct by `{ReferencedRepositoryUrl, PropertyPath}`

Reply:
[here] we have done the aggregation.